.ci/linting.groovy

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env groovy
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'docker && immutable' }
+  environment {
+    HOME = "${env.WORKSPACE}"
+  }
+  options {
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '20', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    rateLimitBuilds(throttle: [count: 60, durationName: 'hour', userBoost: true])
+    quietPeriod(10)
+  }
+  triggers {
+    issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?lint(?:\\W+please)?.*')
+  }
+  stages {
+    stage('Basic Test') {
```

Comment:
Is `Basic Test` a descriptive enough name? As this is a linting pipeline, I can see another name such as `Linting tests` or similar.

wdyt?